app/src/main/java/gateway/soap/ServiceImp.java

```diff
@@ -17,9 +24,37 @@
 		return auth;
 	}
 
+	// user register in auth service, returns an access token
 	@WebMethod public Authorization register(Credentials credentials)
 	{
-		return new Authorization ();
+
+		Authorization auth = new Authorization ();
+
+		String url = EnvironmentVar.AUTH_BASEURL + "register";
+
+		String credentialsString = "{\"username\":\"" + credentials.username +
+								   "\",\"password\":\"" + credentials.password + "\"}";
+
+		try {
+			// Configs and make HTTP POST request to user register.
+			HttpClient client = HttpClient.newHttpClient ();
+
+			HttpRequest request = HttpRequest.newBuilder ()
+									  .uri (URI.create (url))
+									  .POST (BodyPublishers.ofString (credentialsString))
+									  .uri (URI.create (url))
+									  .header ("Content-Type", "application/json")
+									  .build ();
+
+			HttpResponse<String> response =
+				client.send (request, HttpResponse.BodyHandlers.ofString ());
+
+			auth.token = response.body ();
+		} catch (IOException | InterruptedException e) {
+			e.printStackTrace ();
```

Comment:
On error, fill the appropiate fields `success` and `message` from [StatusRes](https://github.com/hawks-atlanta/gateway-java/blob/dev/app/src/main/java/gateway/soap/response/StatusRes.java) which [SessionRes](https://github.com/hawks-atlanta/gateway-java/blob/dev/app/src/main/java/gateway/soap/response/SessionRes.java) inherits.